Momento/Incubating/README.md

```diff
@@ -16,26 +19,37 @@ class Driver
         MomentoSdk.Incubating.SimpleCacheClient scc = new MomentoSdk.Incubating.SimpleCacheClient(
             authToken: "MYTOKEN", defaultTtlSeconds: 60);
 
-        // Set some values
-        CacheDictionarySetResponse sr = scc.DictionarySet(cacheName: "my-cache", dictionaryName: "my-dictionary",
+        // Set a value
+        scc.DictionarySet(cacheName: "my-cache", dictionaryName: "my-dictionary"
+            key: "my-key", value: "my-value", ttlSeconds: 60, refreshTtl: false);
+
+        // Set multiple values
+        scc.DictionaryMultiSet(cacheName: "my-cache", dictionaryName: "my-dictionary",
             new Dictionary<string, string>() {
                 { "key1", "value1" },
                 { "key2", "value2" },
                 { "key3", "value3" }
-        });
-        string dictionaryName = sr.DictionaryName();
-        Dictionary<string, CacheDictionaryValue> dictionary = sr.Dictionary();
+        }, refreshTtl: false);
 
         // Get a value
-        CacheDictionaryGetResponse gr = scc.DictionaryGet(cacheName: "my-cache", dictionaryName: "my-dictionary", key: "key1");
-        CacheGetStatus status = gr.Status;
-        string value = gr.String();
+        CacheDictionaryGetResponse gr = scc.DictionaryGet(cacheName: "my-cache",
+            dictionaryName: "my-dictionary", key: "key1");
```

Comment:
this totally a style nit but would probably write this part like this vs on just 2 lines.
```csharp
CacheDictionaryGetResponse gr = scc.DictionaryGet(
  cacheName: "my-cache",
  dictionaryName: "my-dictionary", 
  key: "key1"
);
```

Idk why the staggered props over 2 lines always stands out to me :)

Reply:
Fixed